test/mock-request.js

```diff
@@ -9,23 +9,25 @@ var expect = require('expect');
 function HttpBackend() {
     this.requests = [];
     this.expectedRequests = [];
-    var self = this;
+    const self = this;
     // the request function dependency that the SDK needs.
     this.requestFn = function(opts, callback) {
-        var realReq = new Request(opts.method, opts.uri, opts.body, opts.qs);
-        realReq.callback = callback;
-        console.log("HTTP backend received request: %s %s", opts.method, opts.uri);
-        self.requests.push(realReq);
+        const req = new Request(opts, callback);
+        console.log("HTTP backend received request: %s", req);
+        self.requests.push(req);
 
-        var abort = function() {
-            var idx = self.requests.indexOf(realReq);
+        const abort = function() {
+            const idx = self.requests.indexOf(req);
             if (idx >= 0) {
-                console.log("Aborting HTTP request: %s %s", opts.method, opts.uri);
+                console.log("Aborting HTTP request: %s %s", opts.method,
+                            opts.uri);
```

Comment:
oh, it just came with the c&p from js-sdk. Where it probably did happen to reduce line length.